module/documents/actor/actor.mjs

```diff
@@ -3453,6 +3454,29 @@ export default class Actor5e extends SystemDocumentMixin(Actor) {
 
   /* -------------------------------------------- */
 
+  /**
+   * Handle applying/removing the bloodied status.
+   * @param {DocumentModificationContext} options  Additional options supplied with the update.
+   * @returns {Promise<ActiveEffect>|void}
+   */
+  updateBloodied(options) {
+    const hp = this.system.attributes?.hp;
+    if ( !hp?.effectiveMax || (game.settings.get("dnd5e", "bloodied") === "none") ) return;
+
+    const effect = this.effects.get(ActiveEffect5e.ID.BLOODIED);
+    if ( hp.value > hp.effectiveMax * CONFIG.DND5E.bloodied.threshold ) return effect?.delete();
```

Comment:
I think we should add a check for 0 HP here too. Meaning if the actor has 0 HP then remove the bloodied effect. Doesn't make sense to me to be bloodied if you're already unconscious or dead.

Reply:
Nothing about the definition of bloodied implies you would lose it at 0 HP. A character at 0 HP, whether they are stable or not, could very well be subjected to an effect where their bloodied condition may be relevant.